redis/rediscache.go

```diff
@@ -0,0 +1,39 @@
+package rediscache
```

Comment:
Package name is `rediscache, but folder is `redis`.

The folder should `rediscache` as well, for consistency with the other existing packages:

- diskcache
- leveldbcache
- memcache